README.md

```diff
@@ -11,7 +11,7 @@ The recommended method of installation is [through composer](http://getcomposer.
 ```JSON
 {
     "require": {
-        "pschwisow/phergie-irc-plugin-react-url-shorten": "dev-master"
+        "pschwisow/phergie-irc-plugin-react-url-shorten": "^1"
```

Comment:
It's probably an odd use case, and if other parts of the Phergie ecosystem are using the new syntax, then a newer version of composer will be required anyway. Mergie as-is.